Makefile

```diff
@@ -106,7 +106,7 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 .PHONY: controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0)
+	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2)
```

Comment:
Why do we need to upgrade controller-gen from 0.7.0 to 0.9.2?
This Makefile is generated when `kuberbuilder init` and usually we don't need to update it?

Reply:
I will submit a separate patch to update controller-gen.